SEImplementation/SEImplementation/Output/LdacOutput.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * LdacOutput.h
+ *
+ *  Created on: Feb 10, 2022
+ *      Author: mschefer
+ */
+
+#ifndef _SEIMPLEMENTATION_OUTPUT_LDACOUTPUT_H_
+#define _SEIMPLEMENTATION_OUTPUT_LDACOUTPUT_H_
+
+#include "Table/FitsWriter.h"
+
+#include "SEImplementation/Output/FlushableOutput.h"
+
+namespace SourceXtractor {
+
+class LdacOutput : public FlushableOutput {
+
+public:
+  LdacOutput (const std::string& filename, SourceToRowConverter source_to_row, size_t flush_size)
+    : FlushableOutput(source_to_row, flush_size), m_filename(filename), m_part_nb(0), m_rms(0), m_gain(0) {
+  }
+
+  void nextPart() override {
+    // TODO does it make sense to implement for LDAC?
```

Comment:
From what I can find, there are not multiple tables for LDAC (?)
By the way, `m_part_nb` isn't used, is it?